src/CoreMedia/CMSync.cs

```diff
@@ -699,6 +812,57 @@ public static bool MightDrift (CMClockOrTimebase clockOrTimebaseA, CMClockOrTime
 
 			return CMSyncMightDrift (clockOrTimebaseA.Handle, clockOrTimebaseB.Handle);
 		}
+
+		[Watch (6,0), TV (9,0), Mac (10,11), iOS (9,0)]
+		[DllImport(Constants.CoreMediaLibrary)]
```

Comment:
```suggestion
		[DllImport (Constants.CoreMediaLibrary)]
```

and all the other ones too.